tests/std/tests/GH_000625_vector_bool_optimization/test.cpp

```diff
@@ -213,8 +215,48 @@ bool test_count() {
     return true;
 }
 
+// Also test the behavior of a huge vector<bool, A> whose size is greater than SIZE_MAX, which is pratical on 32-bit
+// platforms.
+template <class T>
+struct huge_allocator {
+    huge_allocator() = default;
+    template <class U>
+    constexpr huge_allocator(const huge_allocator<U>) noexcept {}

```

Comment:
Although it doesn't really matter, I would prefer to see a const reference here.